controllers/configurationpolicy_controller.go

```diff
@@ -767,7 +789,8 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 		if plc.ObjectMeta.DeletionTimestamp != nil {
 			log.V(1).Info("Config policy has been deleted, handling child objects")
 
-			failures := r.cleanUpChildObjects(plc)
+			// At here, relatedObjects is always []
+			failures := r.cleanUpChildObjects(plc, relatedObjects)
```

Comment:
Maybe this is clearer?
```suggestion
			failures := r.cleanUpChildObjects(plc, nil)
```

Reply:
good